docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -0,0 +1,62 @@
+# Setting up a GoQuorum qlight node
+
+A qlight client node requires a full node configured to act as a qlight server.
+The server node is usually set up to support Multiple Private States (MPS), with the qlight client set up to use a Private State Identifier (PSI) which is managed by the server node.
+
+## Configure qlight client
+
+The qlight client is configured using the following command line options:
+
+- `--qlight.client`: This marks the node as a qlight client.
+- `--qlight.client.psi`: This specifies the PSI which this client will support (default = "private").
+- `--qlight.client.serverNode`: The Node Id of the server node.
+- `--qlight.client.serverNodeRPC`: The RPC URL of the server node.
+
+Additionally, if the server node has the RPC API secured using TLS, then the qlight client requires the following:
+
+- `--qlight.client.rpc.tls`: Use TLS when forwarding RPC API calls.
+- `--qlight.client.rpc.tls.insecureskipverify`: Skip TLS verification.
+- `--qlight.client.rpc.tls.cacert`: The qlight client certificate authority.
+- `--qlight.client.rpc.tls.cert`: The qlight client certificate.
+- `--qlight.client.rpc.tls.key`: The qlight client certificate private key.
+
+## Configure server node
+
+The qlight server is configured using the following command line options:
+
+- `--qlight.server`: This marks the node as a qlight server.
+- `--qlight.server.p2p.port`: The RPC listening port.
+- `--qlight.server.p2p.maxpeers`: The maximum number of qlight clients that are supported.
+
+### Network IP restriction
+
+This restricts communication to specified IP networks (CIDR masks).
+The network mask is specified on the qlight server using the command line option `--qlight.server.p2p.netrestrict`.
+
+### File based permissioning
+
+File based permissioning allows qlight clients to be checked against a permissioned list and a disallowed list.
+It is enabled on the server node using `--qlight.server.p2p.permissioning`.
+
+The default files are `permissioned-nodes.json` and `disallowed-nodes.json`.
+However, a file prefix can be specified using `--qlight.server.p2p.permissioning.prefix`, in which case the filename will be: the prefix, followed by a hyphen, followed by the default file name.
+
+## Using the enterprise authorization protocol integration
+
+This leverages the security model described under [JSON-RPC security](json-rpc-api-security.md#enterprise-authorization-protocol-integration) to only allow authenticated clients to connect to the server.
+
+An access token must be obtained on the qlight client by authenticating with the authorization server.
+The token is then passed on the qlight client command line using `--qlight.client.token`.
+
```

Comment:
When using [JSON-RPC security](json-rpc-api-security.md#enterprise-authorization-protocol-integration) an access token must be provided to communicate to the qlight server. How the token is obtained and refreshed is down to the node operator. 

The token is passed on the qlight client on the command line using `--qlight.client.token`.or it can be update using ???

@nicolae-leonte-go to update :)